specifyweb/frontend/js_src/lib/components/SchemaConfig/Format.tsx

```diff
@@ -314,3 +322,48 @@ function WebLinkEditing({
     </>
   ) : null;
 }
+
+function FieldFormatterEditing({
+  table,
+  value,
+  schemaData,
+}: {
+  readonly table: SpecifyTable;
+  readonly value: string | null;
+  readonly schemaData: SchemaData;
+}): JSX.Element | null {
+  const index = schemaData.uiFormatters
+    .filter((formatter) => formatter.tableName === table.name)
+    .findIndex(({ name }) => name === value);
```

Comment:
(not part of any review - was just curious about it)

This is equivalent to 
```
  const index = schemaData.uiFormatters
    .findIndex(({tableName, name}) => tableName === table.name && name == value)
```

Is there a benefit of using the approach in the PR? Will the second part be faster? - maybe JS engine is smart enough to combine the predicates - SQL is (too much SQL in the past days!)

Reply:
these are not equivalent
if we do only .findIndex(), we get the index in the original array
by filtering first, we get index in the filtered down array (i.e index of a formatter for that particular table, rather than among all tables)

Reply:
Ah I see, I missed that array will be different

Reply:
haha, looks like I shouldn't have done the `.filter()` here after all, as that caused https://github.com/specify/specify7/pull/5075#pullrequestreview-2194509397

the issue is that when computing indexes for record formatters or web links, I don't filter by table - but I forgot about that. And since UI Formatters reuse a lot of code, they should not filter by table either when computing the index